src/Apps/Feature/Components/FeatureFeaturedLink/FeatureFeaturedLink.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Box, Sans, Serif, Image, Flex } from "@artsy/palette"
+
+const Container = styled.a`
```

Comment:
Maybe try `StyledLink` from palette? 

But generally, if the link navigates to another page internal to Artsy.net, we should use a `RouterLink`. Can do `styled(StyledLink)` to apply additional styles, etc.